caseStudies/lib-server.ts

```diff
@@ -0,0 +1,25 @@
+import fs from 'fs';
+import { join } from 'path';
+
+const CASE_STUDY_ARTICLE_FOLDER = 'caseStudies';
+
+const caseStudiesDir = join(process.cwd(), CASE_STUDY_ARTICLE_FOLDER);
+
+type GetTitles = () => string[];
+
+const getTitles: GetTitles = () => fs
```

Comment:
not separate type is probably simpler
const onst getTitles = (): string[] =>{
//
}